### PR TITLE
Jakarta Concurrency with Jakarta Enterprise Beans

### PR DESCRIPTION
**Pull request by njr-11**
Update Concurrency EJB test to run with Jakarta Enterprise Beans.  I was going to switch the bucket entirely to Jakarta, until the tests surfaced a missing autofeature for Jakarta.  If this is the only bucket that runs that code path, then it could be valuable to have the test running against both Java EE and Jakarta EE, to guard against regressions to this autofeature.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.ws.concurrent_fat_ejb/fat/src/com/ibm/ws/concurrent/ejb/fat/ConcurrentEJBTest.java [hunk 1]
```diff
@@ -38,12 +42,23 @@
...
+                    .andWith(new JakartaEE9Action());
+
     @Server("com.ibm.ws.concurrent.fat.ejb")
     @TestServlet(servlet = ConcurrentFATServlet.class, path = APP_NAME)
     public static LibertyServer server;
 
     @BeforeClass
     public static void setUp() throws Exception {
+        ServerConfiguration config = server.getServerConfiguration();
+        if (config.getFeatureManager().getFeatures().contains("servlet-5.0") && config.getFeatureManager().getFeatures().remove("ejblite-3.2")) {
```
Comment (aguibert) [inline dev/com.ibm.ws.concurrent_fat_ejb/fat/src/com/ibm/ws/concurrent/ejb/fat/ConcurrentEJBTest.java:57]: Eventually this check will evaluate to `false` once `ejblite-4.0` is added to `JakartaEEE9Action`, so instead lets use this more direct check:
```java
if (JakartaEE9Action.isActive()) {
  // ...
}
```

**Comment by njr-11** [comment]
#build
#run-libby-bot

File: dev/com.ibm.ws.concurrent_fat_ejb/fat/src/com/ibm/ws/concurrent/ejb/fat/ConcurrentEJBTest.java (hunk 1 above)
Comment (njr-11) [reply dev/com.ibm.ws.concurrent_fat_ejb/fat/src/com/ibm/ws/concurrent/ejb/fat/ConcurrentEJBTest.java]: It turns out the whole issue the test was trying to work around here (jndi-1.0 present in ejblite-3.2 but absent from ejblite-4.0) is no longer the case, so this code back can be deleted entirely.  I've added a commit for that.

**Comment by njr-11** [comment]
Build was good, but there was a merge conflict, so submitting a new build:
#build
#run-libby-bot

**Pull request merged**